index.js

```diff
@@ -22,7 +22,6 @@ var manifest = {
   links: 'source',
   add: 'async',
   publish: 'async',
-  getAddress: 'sync',
```

Comment:
interestingly, this was not in `v19.1.1`

I've been stepping back through time looking for getAddress references: 
```
$ git checkout HEAD~1 && ack getAddress
```

Reply:
adding this back in does not seem to help with the `ssb-server` problem I'm currently having which complains this method doesn't exist